MANIFEST.in

```diff
@@ -9,3 +9,5 @@ include src/dal_select2/static/autocomplete_light/vendor/select2/dist/css/select
 include src/dal/static/autocomplete_light/jquery.init.js
 include src/dal/static/autocomplete_light/autocomplete.init.js
 include src/dal/static/admin/js/jquery.init.js
+recursive-include src/dal_select2/static/autocomplete_light/vendor/select2
```

Comment:
Seems like this makes lines 7 and 8 unnecessary, in this case could we perhaps remove them ?